.travis.yml

```diff
@@ -18,7 +18,7 @@ env:
     # Extension testing
 
     # Package it up to a VSIX
-    - nodejs_version: "7.4.0"
+    - nodejs_version: "8.11.3"
```

Comment:
For reference - As of right now, the VSCode tests use Node 8.9.1 (https://vscode.visualstudio.com/VSCode/_build/results?buildId=4595&view=logs) so 8.11.3 should be fine.